### PR TITLE
fix(filter) filter should remain active after we edit a grade

### PR DESCRIPTION
**Pull request by jansenk**
after editing a grade, rather than doing a blank lookup we should use the current filter

https://openedx.atlassian.net/browse/EDUCATOR-3769

### CODE REVIEW

File: src/components/Gradebook/index.jsx [hunk 1]
```diff
@@ -59,7 +59,11 @@ export default class Gradebook extends React.Component {
           earned_graded_override: this.state.updateVal,
         },
       },
-    ]);
+    ],
+    this.state.filterValue,
```
Comment (schenedx) [inline src/components/Gradebook/index.jsx]: The indentation here doesn't look right. Please correct.

File: src/data/actions/grades.js [hunk 2]
```diff
@@ -132,14 +132,14 @@ const fetchPrevNextGrades = (endpoint, cohort, track) => (
...
-const updateGrades = (courseId, updateData) => (
+const updateGrades = (courseId, updateData, searchText, cohort, track) => (
   (dispatch) => {
     dispatch(gradeUpdateRequest());
     return LmsApiService.updateGradebookData(courseId, updateData)
       .then(response => response.data)
       .then((data) => {
         dispatch(gradeUpdateSuccess(data));
-        dispatch(fetchGrades(courseId, null, null, true));
+        dispatch(fetchMatchingUserGrades(courseId, searchText, cohort, track));
```
Comment (schenedx) [inline src/data/actions/grades.js]: What if we did not have any search texts in the fields, would this still work?
Basically, if the grade adjustment operated on top of default all view, would this change cause breakage?

Comment (jansenk) [reply src/data/actions/grades.js]: it shouldn't break anything. it calls LmsApiService.fetchGradebookData which handles null arguments correctly

Comment (schenedx) [reply src/data/actions/grades.js]: Could you please test that case to make sure?

Comment (jansenk) [reply src/data/actions/grades.js]: seems to be fine!

File: src/components/Gradebook/index.jsx (hunk 1 above)
Comment (jansenk) [reply src/components/Gradebook/index.jsx]: I'm not sure what you mean. What should I change here?

Comment (schenedx) [reply src/components/Gradebook/index.jsx]: So, the ] should be combined with the } above. Then you can have the values indented a "tab" in.
Anyways. Not a big deal.

**Pull request merged**